### PR TITLE
Change to jsoup

### LINKED ISSUE #60: Make Gandalf work
**ONLY IN DOCKER:** 
Gandalf scraps only few pages or none at all. Bonito starts to fail sometimes.

### PR DESCRIPTION
**Pull request by deetree**
Since Selenium used to hang often, the scrappers were rewritten using Jsoup.

It should be discussed and checked whether the new version works as intended and which one to keep.

Closes #60

### CODE REVIEW

**Comment by KeshwaY** [comment]
Remove everything from the `jib` folder as it is not needed anymore.

**Comment by KeshwaY** [comment]
Remove `selenium` and `webdriver` dependencies.

**Comment by KeshwaY** [comment]
Also fix test coverage.

**Comment by KeshwaY** [comment]
Anyway good job with the fix, it works in docker 👍🏻 .

**Comment by deetree** [comment]
> Remove everything from the `jib` folder as it is not needed anymore.

> Remove `selenium` and `webdriver` dependencies.

This should be done in PR related to #62

File: src/test/java/robot/JsoupScrappersIT.java [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        sa.assertFalse(call.books().isEmpty(), "There should be some books scrapped");
+        sa.assertAll();
+    }
+
+    private Document getDocument(String webpage) throws IOException {
+        URL url = Thread.currentThread().getContextClassLoader().getResource(webpage);
+        File file = new File(Objects.requireNonNull(url).getPath());
+        return Jsoup.parse(file, "utf-8");
+    }
+}
```
Comment (KeshwaY) [inline src/test/java/robot/JsoupScrappersIT.java:85]:
```suggestion
}

```

File: src/main/java/robot/Bookstore.java [hunk 2]
```diff
@@ -1,8 +1,8 @@
 package robot;
 
 public enum Bookstore {
-    GANDALF(new GandalfScrapper()),
-    BONITO(new BonitoScrapper());
+    GANDALF(new GandalfScrapperJsoup("https://www.gandalf.com.pl/promocje/bcb/")),
```
Comment (KeshwaY) [inline src/main/java/robot/Bookstore.java:4]:
```suggestion
    GANDALF(new GandalfScrapperJsoup("https://www.gandalf.com.pl/promocje/bcb")),
```

**Review by KeshwaY — CHANGES_REQUESTED** [review]

**Pull request merged**